uws/UWS/models.py

```diff
@@ -10,6 +10,25 @@ def _parse_bool(self, value):
             return False
         return value
 
+    def _find_ns_uws(self, namespaces):
+        # Search for namespace that contains "http://www.ivoa.net/xml/UWS/"
+        # (usually, there is a version attached, which we ignore here),
+        # for using it (including {}) as uws_ns in the
+        # find/findall-functions everywhere below.
+        ns = None
+        for p in namespaces.keys():
```

Comment:
actually you can be more pythonic here with:

`for namespace, key in namespaces.itteritems():`

then you don't need the value line below